contents.json

```diff
@@ -5776,6 +5776,11 @@
       "category": "quality",
       "description": "A code library and command-line formatting tool for reformatting Swift code.",
       "homepage": "https://github.com/nicklockwood/SwiftFormat"
+    }, { 
+      "title": "SwiftShell",
+      "category": "command-line",
+      "description": "A Swift framework for shell scripting.",
```

Comment:
Hi @IceCreamCremo, let's change the description to "A library for creating command-line applications and running shell commands". Due to this point "The description usually should not include "written in Swift" or "in Swift" as everything on the list would have that" in [Contribution Guidelines](https://github.com/matteocrippa/awesome-swift/blob/master/.github/CONTRIBUTING.md).

Reply:
fixed